guides/common/modules/proc_removing-lifecycle-environments-from-smart-proxy-by-using-web-ui.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="removing-lifecycle-environments-from-smart-proxy-by-using-web-ui"]
+= Removing lifecycle environments from {SmartProxyServer} by using {ProjectWebUI}
+
+[role="_abstract"]
+When lifecycle environments are no longer relevant to the host system or environments are added incorrectly to {SmartProxyServer}, you can remove the lifecycle environments from {SmartProxyServer}.
```

Comment:
```suggestion
When lifecycle environments are no longer relevant to hosts or environments are added incorrectly to {SmartProxyServer}, you can remove the lifecycle environments from {SmartProxyServer}.
```

I suggest to go with plural "hosts" in favor of "host systems" to unify terminology.